api/management/swagger-api_management.yaml

```diff
@@ -968,6 +968,46 @@ paths:
           description: successful operation
         400:
           description: invalid information provided  (invalid client identifier or redirect URI not allowed for this client)
+  /realms/{realm}/admin-configuration:
+    get:
+      tags:
+      - Configuration
+      summary: Get the current admin configuration
```

Comment:
of the current or the one specified in param ?

Reply:
Updating: "Get the admin configuration of the specified realm"